### ISSUE TITLE
Add support for right-click and pasting code

### ISSUE THREAD

**Issue by rahul-sharma-uipath** [andreaswilli/react-verification-input#77]
Currently, if you right-click the component, it renders the context menu for a `div` which doesn't allow pasting the code.

The workaround is Ctrl + V for pasting the code but right-click and paste would be preferred.

<img width="402" alt="image" src="https://user-images.githubusercontent.com/50849907/185675604-7f6e761f-40bc-4152-9b61-0934b426eb24.png">

**Comment by andreaswilli** [comment]
Right, that's a limitation of the current implementation. As you mentioned, the visible parts are not input elements, thus there is no paste option. I can try to find a way to support right-click pasting anyway.
If you have an idea, let me know.

**Comment by rahul-sharma-uipath** [comment]
@andreaswillis what if you placed the `input` element "above" the first div instead of hiding it from the screen?

**Comment by andreaswilli** [comment]
That could be a solution. Gonna give it a try.

**Comment by andreaswilli** [comment]
I just released `v3.1.0` with right-click support.

The invisible input is now placed over the visible elements which also fixes some other issues on iOS.

**Comment by rahul-sharma-uipath** [comment]
@andreaswilli one thing I noticed from this - the default input component has this box around it as shown below
![image](https://user-images.githubusercontent.com/50849907/189425541-9950e7ed-79a3-460b-beb1-780ed81f6182.png)

however, when I try to pass `className` via the `inputProps` e.g. 
```
        <VerificationInput
            inputProps={{
                disabled: loading,
                className: classes.input,
                ...inputProps,
            }}
        />
```
then the following happens - 
![image](https://user-images.githubusercontent.com/50849907/189425720-84179bb8-1cec-49e4-930c-12c057767946.png)

the reason appears to be that the `.vi` class as shown below, gets overwritten by the prop being passed in. I think it would make sense to combine the classes in some way to ensure that the vi class is always there and then additional classes are added on top of that.

without override -
![image](https://user-images.githubusercontent.com/50849907/189425861-9f68c334-ddc1-4684-a630-60235439e822.png)

with override - 
![image](https://user-images.githubusercontent.com/50849907/189426028-42afaca3-1b39-4bea-90b5-df40a385e95a.png)

**Comment by andreaswilli** [comment]
@rahul-sharma-uipath 
Combining the default and custom class names sounds reasonable.

But I'm curious where this box comes from. Haven't seen this on any device I tested.
What OS/browser are you using? Also, are you using any special settings like high contrast mode?

**Comment by rahul-sharma-uipath** [comment]
it repros fairly consistently for me with no customizations - see https://codesandbox.io/s/nervous-sound-30qstc?file=/src/App.js

<img width="602" alt="image" src="https://user-images.githubusercontent.com/50849907/189503490-f9368560-fab4-4669-836a-12e203f13d30.png">

I'm using an M1 Macbook Pro macOS 12.4 on Chrome Version 105.0.5195.102 (arm64) with no special settings

**Comment by andreaswilli** [comment]
Thanks for the codesandbox. I'm getting the same result now.

On the demo website the border is removed by tailwind, that's why I didn't see it.

**Comment by andreaswilli** [comment]
@rahul-sharma-uipath Released `3.1.1` with some fixes. Let me know if the issues are resolved.
